hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -154,6 +154,9 @@ def copy(self, default=None):
     @api.model_create_multi
     def create(self, vals_list):
         res = super().create(vals_list)
+        # If we install a module with data, we need to convert the calendar to calendar_ids
+        if self.env.context.get("install_mode") and "calendar_ids" not in vals_list:
```

Comment:
Shouldn't this be done always instead of only in install mode?

Reply:
Makes sense.

Reply:
BTW I just spotted a bug because `vals_list` is a list and not a dict.